modules/destroy-hc-ibmpower-cli.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-destroy/hcp-destroy-ibmpower.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="destroy-hc-ibmpower-cli_{context}"]
+= Destroying a hosted cluster on {ibm-power-title} by using the CLI
+
+You can use the `hcp` command-line interface (CLI) to destroy a hosted cluster on {ibm-power-title}.
+
+.Procedure
+
+* Delete the hosted cluster and its backend resources by running the following command:
```

Comment:
> backend resources
Can you please elaborate on what and all covered by this? Because this would only clean up things on management cluster. It will not delete the actual worker VMs or other resources like DNS records.

Reply:
Given that we are using `destroy cluster` command. If it's fine, I will exclude using "the backend resources". Also, will catch up on slack regarding deleting VMS and DNS records.

Reply:
Sure we can do that for now.